.ci/aarch64/lib_install_qemu_aarch64.sh

```diff
@@ -45,34 +45,28 @@ install_packaged_qemu() {
 }
 
 build_and_install_qemu() {
-        QEMU_REPO=$(get_version "assets.hypervisor.qemu.url")
+        QEMU_REPO_URL=$(get_version "assets.hypervisor.qemu.url")
         # Remove 'https://' from the repo url to be able to clone the repo using 'go get'
-        QEMU_REPO=${QEMU_REPO/https:\/\//}
+        QEMU_REPO=${QEMU_REPO_URL/https:\/\//}
         PACKAGING_REPO="github.com/kata-containers/packaging"
         QEMU_CONFIG_SCRIPT="${GOPATH}/src/${PACKAGING_REPO}/scripts/configure-hypervisor.sh"
 
-        go get -d "${QEMU_REPO}" || true
-        go get -d "$PACKAGING_REPO" || true
+        git clone --branch "$CURRENT_QEMU_BRANCH" --single-branch "${QEMU_REPO_URL}" "${GOPATH}/src/${QEMU_REPO}"
```

Comment:
if we do go with `git clone`, and afaik we are just building from the HEAD of the code, maybe we can do this with a `--depth 1` to make a shallow clone, and save us time and bandwidth? :-)
iirc, there may be submodules in the qemu tree as well, so we may also want/need `--shallow-submodules` as well?

Reply:
Hi~ @grahamwhaley @devimc I have addressed your comments in another PR #1555, ptal. ;)